res/css/views/messages/_MessageActionBar.scss

```diff
@@ -29,7 +29,8 @@ limitations under the License.
     right: 8px;
     user-select: none;
     // Ensure the action bar appears above over things, like the read marker.
-    z-index: 1;
+    z-index: 10;
+    //Edited the z-index so that the action bar appears above avatar in small screens
```

Comment:
This comment should be above the z-index change, alongside the other one. Suggested re-wording:

> Ensure the action bar appears above over things like the read marker and sender avatar (for small screens)

Reply:
Done in 766d74b73179c39b229288044b53762c56e473a9